--- ./glib/Makefile.am.link	2010-05-31 14:35:00.000000000 +0200
+++ ./glib/Makefile.am	2010-05-31 14:37:05.000000000 +0200
@@ -20,7 +20,7 @@
 endif
 
 libpyglib_2_0_@PYTHON_BASENAME@_la_CFLAGS = $(GLIB_CFLAGS)
-libpyglib_2_0_@PYTHON_BASENAME@_la_LIBADD = $(GLIB_LIBS) $(FFI_LIBS)
+libpyglib_2_0_@PYTHON_BASENAME@_la_LIBADD = $(GLIB_LIBS) $(FFI_LIBS) `$(PYTHON)-config --libs`
 libpyglib_2_0_@PYTHON_BASENAME@_la_SOURCES = 	\
 	pyglib.c		\
 	pyglib.h		\
--- ./glib/Makefile.in.link	2010-05-31 14:37:11.000000000 +0200
+++ ./glib/Makefile.in	2010-05-31 14:37:20.000000000 +0200
@@ -262,7 +262,7 @@
 pyglib_LTLIBRARIES = _glib.la
 common_ldflags = -module -avoid-version $(am__append_1)
 libpyglib_2_0_@PYTHON_BASENAME@_la_CFLAGS = $(GLIB_CFLAGS)
-libpyglib_2_0_@PYTHON_BASENAME@_la_LIBADD = $(GLIB_LIBS) $(FFI_LIBS)
+libpyglib_2_0_@PYTHON_BASENAME@_la_LIBADD = $(GLIB_LIBS) $(FFI_LIBS) `$(PYTHON)-config --libs`
 libpyglib_2_0_@PYTHON_BASENAME@_la_SOURCES = \
 	pyglib.c		\
 	pyglib.h		\
--- ./gio/Makefile.am.link	2010-05-31 14:40:07.000000000 +0200
+++ ./gio/Makefile.am	2010-05-31 14:41:10.000000000 +0200
@@ -84,7 +84,7 @@
 unix.c: $(GIOUNIX_DEFS) unix.override
 unix_la_CFLAGS = $(GIOUNIX_CFLAGS)
 unix_la_LDFLAGS = $(common_ldflags) -export-symbols-regex initunix
-unix_la_LIBADD = $(GIOUNIX_LIBS)
+unix_la_LIBADD = $(GIOUNIX_LIBS) `$(PYTHON)-config --libs`
 unix_la_SOURCES = unixmodule.c
 nodist_unix_la_SOURCES = unix.c
 if BUILD_GIOUNIX
--- ./gio/Makefile.in.link	2010-05-31 14:41:15.000000000 +0200
+++ ./gio/Makefile.in	2010-05-31 14:41:33.000000000 +0200
@@ -321,7 +321,7 @@
 GIOUNIX_DEFS = unix.defs unix-types.defs unix.override
 unix_la_CFLAGS = $(GIOUNIX_CFLAGS)
 unix_la_LDFLAGS = $(common_ldflags) -export-symbols-regex initunix
-unix_la_LIBADD = $(GIOUNIX_LIBS)
+unix_la_LIBADD = $(GIOUNIX_LIBS) `$(PYTHON)-config --libs`
 unix_la_SOURCES = unixmodule.c
 nodist_unix_la_SOURCES = unix.c
 all: all-am
--- ./gi/Makefile.am.link	2011-03-23 06:29:56.000000000 +0100
+++ ./gi/Makefile.am	2011-03-23 06:30:18.000000000 +0100
@@ -68,7 +68,7 @@
 	-export-symbols-regex "init_gi_cairo|PyInit__gi_cairo"
 _gi_cairo_la_LIBADD = \
 	$(GI_LIBS) \
-	$(PYCAIRO_LIBS)
+	$(PYCAIRO_LIBS) `$(PYTHON)-config --libs`
 _gi_cairo_la_SOURCES = pygi-foreign-cairo.c
 
 
--- ./gi/Makefile.in.link	2011-03-23 06:30:31.000000000 +0100
+++ ./gi/Makefile.in	2011-03-23 06:30:53.000000000 +0100
@@ -396,7 +396,7 @@
 
 _gi_cairo_la_LIBADD = \
 	$(GI_LIBS) \
-	$(PYCAIRO_LIBS)
+	$(PYCAIRO_LIBS) `$(PYTHON)-config --libs`
 
 _gi_cairo_la_SOURCES = pygi-foreign-cairo.c
 pygi_LTLIBRARIES = _gi.la $(am__append_1)
